File: bots/csharpClient/Bots/MyAi.cs
using CsClient.Bots.Internal;

namespace CsClient.Bots
{
    public class MyAi : IBot
    {
        /// <inheritdoc/>
        public string Name { get; }

        private bool _fly = true;

        /// <inheritdoc/>
        public bool Play(PlayState? playState)
        {
            // put your logic here
            return _fly;
        }

        public MyAi()
        {
            // give your bot a super duper cool name
            Name = "YourNameHere";
        }
    }
}

File: bots/typescriptClient/src/bots/myAI.ts
import { BotAI } from './botAI';
import { PlayState } from './data';

export class MyAI implements BotAI {
  private name: string = 'YourBotName';
  private fly: boolean = true;

  play(currentGameState: PlayState): boolean {
    // todo: put your logic code here
    return this.fly;
  }

  getName(): string {
    return this.name;
  }
} 